analysis/deep2_r23_o32/plotting/plot_difference_curvefit.py

```diff
@@ -221,7 +220,12 @@ def plot_difference_twovariable(lR23, lO32, OH, bin_start, bin_end, pdf_file,
                 i_diff = lR23[nn][idx] - LAC[idx]
                 ax.scatter(OH[nn][idx], i_diff, color=ctype[ii],
                            marker=marker[nn], edgecolor='none', alpha=0.5)
-                diff0 += list(lR23[nn][idx] - LAC[idx])
+                if label[nn] == 'Robust Limits':
+                    ax.scatter(OH[nn][idx], i_diff, color=ctype[1],
+                               marker=r'$\rightarrow$',
+                               edgecolor='none', alpha=0.5)
+                if label[ii] != 'Robust Limits':
```

Comment:
You're switching between using `nn` and `ii` for label. Also, labelling should only do once so not sure why it's within a sub-for loop